portality/models/article.py

```diff
@@ -591,6 +596,34 @@ def save(self, *args, **kwargs):
         self._generate_index()
         return super(Article, self).save(*args, **kwargs)
 
+    def get_owner(self):
```

Comment:
If we're going to introduce this method, I'd love to DRY up the code a bit and use it in `portality.bll.services.article` in `is_legitimate_owner`.  That function could call this function, and then handle the exceptions to give the return values that it wants.